streampark-flink/streampark-flink-kubernetes/src/main/scala/org/apache/streampark/flink/kubernetes/ingress/IngressStrategyV1.scala

```diff
@@ -55,11 +55,12 @@ class IngressStrategyV1 extends IngressStrategy {
         val ingress = new IngressBuilder()
           .withNewMetadata()
           .withName(clusterId)
-          .addToAnnotations(buildIngressAnnotations(clusterId).asJava)
+          .addToAnnotations(buildIngressAnnotations(clusterId, nameSpace).asJava)
           .addToLabels(buildIngressLabels(clusterId).asJava)
           .addToOwnerReferences(ownerReference) // Add OwnerReference
           .endMetadata()
           .withNewSpec()
+          .withIngressClassName("nginx")
```

Comment:
I think there is no need to set the ingressClass forcibly. The default value of ingressClass is "". Kubernetes will choose the default ingressClass, which is "nginx" for many Kubernetes instances. However, there are also some special cases, such as the default ingress provided by some cloud providers for Kubernetes not being nginx.

Reply:
THKS, i agree whith you. already fix it